app/BUILD.bazel

```
@@ -663,6 +663,8 @@ kt_android_library(
         "//utility",
         "//utility/src/main/java/org/oppia/android/util/accessibility:prod_module",
         "//utility/src/main/java/org/oppia/android/util/extensions:bundle_extensions",
+        #Replace debug_module with prod_module when building the app in prod mode.
```

```suggestion
        # Replace debug_module with prod_module when building the app in prod mode.
```

Please also add a TODO here on #2432 to set this up properly.

---

Done.